packages/property-grid/public/locales/en/PropertyGrid.json

```diff
@@ -0,0 +1,29 @@
+{
+  "widget-label": "Properties Widget",
+  "context-menu": {
+    "find-similar": {
```

Comment:
oh wait, it is being used, then do we need to oss that pkg too? or do we just allow users to add their own components to the properties widget too?